src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml

```diff
@@ -173,6 +175,29 @@
 
                 </Button>
 
+                <Button Width="Auto" 
+                        Grid.Column="2" 
+                        Click="OnFilterButtonClicked" 
+                        fa:Awesome.Content="Filter"
```

Comment:
What do you think of also adding a tooltip here? to the filter icon if it doesn't exist already?

Reply:
I can apply the same tooltip which I just added also to the context options, but the button I think need some different messaging?

Reply:
@mjkkirschner More tooltips added